review_july (#172)
fiskaltrust/productdescription-de-doc: product-service-description/market-de-fair-use-policy.md

> ### Outlet

Reviewer: ```suggestion
### Outlet (Standort)
```

> Ein Outlet repräsentiert grundsätzlich einen physischen Standort (Filiale oder Niederlassung) des KassenBetreibers. Ein physischer Standort kann jedoch auch durch mehrere, virtuelle Outlets repräsentiert sein, wenn die hier dargestellten Fair Use Regeln nicht mit Hilfe eines einzelnen Outlets eingehalten werden können. 

Reviewer: ```suggestion
Ein Outlet repräsentiert grundsätzlich einen physischen Standort (Filiale oder Niederlassung) des KassenBetreibers. Ein physischer Standort kann jedoch auch durch mehrere, virtuelle Outlets repräsentiert sein, wenn die hier dargestellten [Fair Use Policy](https://docs.fiskaltrust.cloud/de/docs/product-description/germany/products-and-services/fair-use-policy) nicht mit Hilfe eines einzelnen Outlets eingehalten werden können. 
```

### Aktive Queue

> Eine aktive Queue ist eine im Einsatz befindliche Queue, die durch einen Inbetriebnahmebeleg in Betrieb genommen wurde und noch nicht durch einen Außerbetriebnahmebeleg außer Betrieb genommen wurde. Sie repräsentiert ein elektronisches Aufzeichnungssystem mit Kassenfunktion. 

Reviewer: ```suggestion
Eine aktive fiskaltrust.Queue ist eine im Einsatz befindliche Queue, die durch einen Initial [operation receipt / start-receipt](https://docs.fiskaltrust.cloud/de/docs/poscreators/middleware-doc/germany/reference-tables/ftreceiptcase) in Betrieb genommen wurde und noch nicht durch einen [Out of operation receipt / stop-receipt](https://docs.fiskaltrust.cloud/de/docs/poscreators/middleware-doc/germany/reference-tables/ftreceiptcase) außer Betrieb genommen wurde. Sie repräsentiert rechtlich ein fiskalisiertes POS-System. 
```

### Terminals

> Terminals sind an einem elektronischen Aufzeichnungssystem mit Kassenfunktion angeschlossene Eingabegeräte, die keine eigenständige Kassenfunktion implementieren. Im Rahmen des fiskaltrust.Ipos Interfeace werden diese anhand des Feldes `cbTerminalID` identifiziert. 

Reviewer: ```suggestion
Terminals sind an einem POS-System angeschlossene Eingabegeräte, die keine eigenständige Kassenfunktion bereitstellen. Im Rahmen des fiskaltrust.Ipos-Interfeace werden diese anhand des Feldes `[cbTerminalID](https://docs.fiskaltrust.cloud/de/docs/poscreators/middleware-doc/general/data-structures)` identifiziert. 
```

> ### Standortbezogene Produkte der fiskaltrust gmbh

Reviewer: ```suggestion
### Standortbezogene Produkte der fiskaltrust
```

> Standortbezogene Produkte, sind Produkte (bzw. Subscriptions) die pro Outlet erworben werden. Sie werden für die, dem Outlet untergeordneten Aufzeichnungssysteme (Queues) und technischen Sicherheitseinrichtungen (TSE / SCU), eingesetzt (z.B. Sorglos Bundles).

Reviewer: ```suggestion
Standortbezogene Produkte, sind Produkte (bzw. Subscriptions) die pro Outlet erworben werden. Sie werden für die, dem Outlet untergeordneten POS-System (Queues) und technischen Sicherheitseinrichtungen (TSE / SCU), eingesetzt (z.B. Sorglos Bundles).
```

> ## Fair Use Regeln

Reviewer: ```suggestion
## Fair Use Policy
```

> Werden in einem Outlet standortbezogene Produkte der fiskaltrust eingesetzt, so dürfen in dem betroffenen Outlet maximal 25 aktive Queues mit insgesamt maximal 25 Kassen oder Terminals betrieben werden. Zudem dürfen im betroffenen Outlet maximal 150.000 TSE Signaturen pro Kalendermonat angefordert werden.

Reviewer: ```suggestion
Die Fair Use Police für den deutschen Markt ist hier zu finden: [https://docs.fiskaltrust.cloud/de/docs/product-description/germany/products-and-services/fair-use-policy](https://docs.fiskaltrust.cloud/de/docs/product-description/germany/products-and-services/fair-use-policy)
```

Diese Fair Use Regeln können aufgrund spezifischer Eigenschaften einzelner standortbezogener Produkte durch fiskaltrust angepasst werden. Achten Sie deshalb bitte auch auf die konkreten Produktmerkmale.

> ### Empfehlung zur Einhaltung der Fair Use Regel

Reviewer: ```suggestion
### Empfehlung zur Einhaltung der Fair Use Policy
```

> Sollte ein physischer Standort aufgrund höherer Anforderungen nicht durch ein einzelnes Outlet abgedeckt werden können, so steht Ihnen die Möglichkeit zur Verfügung für diesen physischen Standort zusätzliche, virtuelle Outlets, im fiskaltrust fiskaltrust.Portal anzulegen. Die virtuellen Outlets müssen mit der Adresse des physischen Standorts im fiskaltrust fiskaltrust.Portal angelegt werden damit sie diesem zugeordnet werden können. Pro angelegtem Outlet können dann separat die benötigten, standortbezogenen fiskaltrust Produkte erworben werden.

Reviewer: ```suggestion
Sollte ein physischer Standort aufgrund höherer Anforderungen nicht durch ein einzelnes Outlet abgedeckt werden können, so steht Ihnen die Möglichkeit zur Verfügung für diesen physischen Standort zusätzliche, virtuelle Outlets, im fiskaltrust fiskaltrust.Portal anzulegen. Die virtuellen Outlets müssen mit der Adresse des physischen Standorts im [fiskaltrust.Portal](https://portal.fiskaltrust.de/AccountOutlet) angelegt werden damit sie diesem zugeordnet werden können. Dadurch können die benötigten, standortbezogenen fiskaltrust Produkte pro angelegtem Outlet separat erworben werden.
```

### Nichteinhaltung der Fair Use Policy

> Bei Nichteinhaltung der Fair Use Policy behält sich die fiskaltrust das Recht vor, den Betrieb der im betroffenen Outlet verwendeten Produkte und Komponenten einzuschränken, bis sich diese wieder innerhalb des hier beschriebenen Fair-Use-Rahmen befinden.

Reviewer: ```suggestion
Bei Nichteinhaltung der [Fair Use [](Policy](https://docs.fiskaltrust.cloud/de/docs/product-description/germany/products-and-services/fair-use-policy)) behält sich fiskaltrust das Recht vor, den Betrieb der im betroffenen Outlet verwendeten Produkte und Komponenten einzuschränken, bis sich diese wieder innerhalb der geltenden Fair Use Policy befinden.
```
